api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -66,6 +66,7 @@
 @RestController
 public class ProfileController implements ProfileApiDelegate {
   private String ID_VERIFICATION_TEXT = "A new user has requested manual ID verification: ";
+  private String ADMIN_EMAIL = "stable-environment-manual-id-verification-requests@fake-research-aou.org";
```

Comment:
This seems like a perfect example of something that should be in a config rather than code.

Reply:
+1, please add to WorkbenchConfig.

Reply:
Sounds good! For some reason I thought about config and then didn't... I can't remember the reason so it probably wasn't important.